### PR TITLE
join page form enhancement

### PR DESCRIPTION
**Pull request by Neenu-12**
### What is the change?

- Made form fields on join page look usable
- Added outline for the input boxes
- Added a default disabled option for "Your Country" fiels

### Is Development Tested?

- [x] Yes

### Before / After Change Screenshots

For visual or interaction changes. Please provide a video / screenshot.

##### Example:

Before :
before

After :
Screenshot (3)

### CODE REVIEW

File: join.html [hunk 1]
```diff
@@ -181,7 +181,7 @@ <h2 class="progress">Step 1 of 4: Personal Details</h2>
             type="text"
             name="city"
             id="city"
-            placeholder="Enter your city"
+            placeholder="e.g Bangaluru"
```
Comment (ankushdharkar) [inline join.html:184]: Typo in the name here.

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: join.html (hunk 1 above)
Comment (Neenu-12) [reply join.html:184]: @ankushdharkar thanks for the review will make the change 😄

**Comment by Neenu-12** [comment]
@ankushdharkar I have made the required changes please preview them.

File: join.html [hunk 2]
```diff
@@ -207,7 +207,9 @@ <h2 class="progress">Step 1 of 4: Personal Details</h2>
             type="text"
             name="country"
             id="country"
-          ></select>
+          >
+            <option value="" disabled selected>e.g India</option>
+          </select>
```
Comment (RitikJaiswal75) [inline join.html:212]: This Part of the code is not working, I can not see anything already selected

image

**Review by RitikJaiswal75 — CHANGES_REQUESTED** [review]

**Review by Maheima — CHANGES_REQUESTED** [review]
image
1. can we make the width of all the input areas equal?
2. A little padding on the right of the arrow in the select option bar
3. The placeholder text's color of country is different than that of city and state
4. 
image
The background of next button is getting mixed with that of the root background. Can we please have a contrast there for accessibility reasons?

File: join.html (hunk 2 above)
Comment (Neenu-12) [reply join.html:212]: @RitikJaiswal75 It is working fine please have a look
image

**Comment by Neenu-12** [comment]
@Maheima Thanks for the review I'm working on the changes.

**Comment by Neenu-12** [comment]
@Maheima I have made all the changes you suggested please review them.

**Comment by Neenu-12** [comment]
@Maheima Here are the screenshots :-
image

image

image

**Comment by Maheima** [comment]
> @Maheima Here are the screenshots :- image
> 
> image
> 
> image

Thanks for fixing them all. LGTM 👍

**Comment by Neenu-12** [comment]
@Maheima can you guide me what the further steps are so that I get the changes merged ?

File: join.html (hunk 2 above)
Comment (rohan09-raj) [reply join.html:212]: @Neenu-12 Please clear you local storage and then check for default selection

Comment (Neenu-12) [reply join.html:212]: Hi @rohan09-raj  cleared the local storage as you suggested and here's the screenshot of the page :-

image

It's working fine

Comment (rohan09-raj) [reply join.html:212]: Great thanks 💯

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**